pyangbind/lib/serialise.py

```diff
@@ -97,7 +104,7 @@ def default(self, obj, mode='default'):
     if orig_yangt in ["leafref"]:
       return self.default(obj._get()) if hasattr(obj, "_get") \
                                                   else unicode(obj)
-    elif orig_yangt in ["int64", "uint64"]:
+    elif orig_yangt in ["int64", "uint64", "derived64"]:
```

Comment:
What is `derived64` here please?

Reply:
https://github.com/robshakir/pyangbind/blob/master/tests/serialise/ietf-json-serialise/ietf-json-serialise.yang#L31

Reply:
This shouldn't be in the code here --`typedef` can have arbitrary names, so we need to check the underlying types here otherwise for each of the types that an arbitrary YANG schema adds, we'd need to add them here. 

This should likely be handled here: https://github.com/robshakir/pyangbind/blob/master/pyangbind/lib/serialise.py#L140, it's not clear to me why it isn't. Does Python 3 change something about whether we have `int` and `long`?